go/vt/sqlparser/parse_test.go

```diff
@@ -670,6 +670,9 @@ var (
 	}, {
 		input:  "set charset default",
 		output: "set ",
+	}, {
```

Comment:
I think it would make sense to modify `Set.Format` to print both the Exprs and the charset (if non-null) so that this test (and all the ones above it) would more clearly show that the parser is preserving the charset.
https://github.com/tinyspeck/vitess/blame/master/go/vt/sqlparser/ast.go#L579-L587